SingularityUI/app/components/common/table/UITable.jsx

```diff
@@ -452,6 +452,7 @@ UITable.propTypes = {
   keyGetter: PropTypes.func.isRequired,
   children: PropTypes.arrayOf(PropTypes.node).isRequired,
   paginated: PropTypes.bool,
+  isTruelyInfinite: PropTypes.bool, // Will render all rows even if there are 1,000,000
```

Comment:
I'll respond to the second point first. The problem is there are now three pagination options, so a boolean can't encompass them all:
- Paginated (this can be split into api paginated and local paginated, but that's not relevant here)
- Not paginated, but uses infinite scrolling (which is what the requests and tasks tables do)
- Not paginated, renders all rows no matter what (which is introduced in this PR and is what the `My Starred Requests` table will do)

The third pagination type is implemented because we want to render all starred requests for ctrl-f access - which won't happen if the user has more than a certain number if we're using infinite scrolling.

So to be more descriptive with the name, maybe `renderAllRowsImmediately` or something like that would be a better name.